consensus/safety-rules/src/safety_rules.rs

```diff
@@ -136,6 +136,19 @@ impl<T: Payload> SafetyRules<T> {
             Ok(())
         }
     }
+
+    /// This checkes whether the author of one proposal is the validator signer
+    fn verify_author(&self, author: Option<Author>) -> Result<(), Error> {
+        let validator_signer_author = &self.validator_signer.author();
+        let author = author
+            .ok_or_else(|| Error::InvalidProposal("No author found in the proposal".into()))?;
+        if (*validator_signer_author) != author {
```

Comment:
I think if you instead do:
`if validator_signer_author != &author`
you can avoid the double parentheses